### DISCUSSION TITLE
Sharing backend login with frontend

### DISCUSSION THREAD

**Discussion by ghost** [area17/twill#1265] [Q&A] [answered]
Hi, me again!

Is there a way to share backend login information with frontend modules? e.g. If I log in to the admin area, is there a way to retrieve that information in the frontend?

Cheers!
Aaron

**Comment by pboivin** [comment, marked as answer]
Hi @aarontilt, I guess this will depend on your frontend configuration but if you're using standard Blade views on your frontend, you can access the current Twill user like this:
```
$user = Auth::guard('twill_users')->user();
```

You shoud get `null` if the current user is not logged in.

**Reply by ghost** [reply]
Just given that a go, but no dice. My backend is "admin.localhost" at the moment, and my frontend is just "localhost", if that helps (maybe to do with session)?

**Reply by ghost** [reply]
Ah, never mind. Figured it out. I've changed the .env file to include:
```
ADMIN_APP_URL=http://localhost
ADMIN_APP_PATH=admin
```
and that seems to have sorted it

**Reply by pboivin** [reply]
Makes sense, the 2 domains would each have their own PHP session. I think you can share the session cookie between `admin.localhost` and `localhost` with this `.env` setting :
```
SESSION_DOMAIN=.localhost
```
